satellite/mailservice/simulate/linkclicker.go

```diff
@@ -24,38 +24,20 @@ func (clicker *LinkClicker) FromAddress() post.Address {
 
 // SendEmail click all links from email html parts
 func (clicker *LinkClicker) SendEmail(msg *post.Message) error {
+	regx := regexp.MustCompile(`href="([^\s])+"`)
```

Comment:
```
rxLink := regexp.MustCompile(`<a .* href="(.*)">(.*)</a>`
```

Might suffice... maybe needs to avoid newlines. For these avoiding "packages" with "less correct" approaches, I usually add a comment like "// dirty way to find links without pulling in a dependency"... helps others orient why something "more correct wasn't used".

Reply:
I experimented with this a lot and the one you provide picks all the code from first `a` start tag to last closing tag, if multiline is set, or miss cases when there is text or some other tag is inside `a` and single line set to true. Also the link could never include whitespace(or I hope so, it's usually escaped %20 or smth).

Reply:
Yeah, it needs to be set as non-greedy and non-multiline... oh-well let's use the one you have then.